tritium-metrics-jvm/src/main/java/com/palantir/tritium/metrics/jvm/JvmMetrics.java

```diff
@@ -67,7 +67,14 @@ private static void registerAttributes(InternalJvmMetrics metrics) {
         Map<String, Metric> jvmAttributes = new JvmAttributeGaugeSet().getMetrics();
         metrics.attributeName(gauge(jvmAttributes, "name"));
         metrics.attributeVendor(gauge(jvmAttributes, "vendor"));
-        metrics.attributeUptime(gauge(jvmAttributes, "uptime"));
+        metrics.attributeUptime()
+                .javaSpecificationVersion(System.getProperty("java.specification.version", "null"))
```

Comment:
I think `unknown` has more meaning than `null` when we consume the results.
```suggestion
                .javaSpecificationVersion(System.getProperty("java.specification.version", "unknown"))
```